rs/http_utils/BUILD.bazel

```
@@ -37,3 +38,14 @@ rust_test(
     crate = ":http_utils",
     deps = DEPENDENCIES + DEV_DEPENDENCIES,
 )
+
+rust_test(
+    name = "http_utils_long_test",
```

I wouldn't have `long_test` in the name since we might want to remove the `long_test` at some point and then it would be good to keep using the same name for the test. So what about just:
```suggestion
    name = "http_utils_test",
```

---

Other unit tests are called`http_utils_test`. I would have to somehow distinguish the two